tests/test_astrometryTask.py

```diff
@@ -211,9 +224,59 @@ def makeSourceCat(self, distortedWcs):
             src.set(sourceCentroidKey, refObj.get(refCentroidKey))
             src.set(sourceInstFluxKey, refObj.get(refFluxRKey))
             src.set(sourceInstFluxErrKey, refObj.get(refFluxRKey)/100)
+
         return sourceCat
 
 
+class TestMagnitudeOutliers(lsst.utils.tests.TestCase):
+    def testMagnitudeOutlierRejection(self):
+        """Test rejection of magnitude outliers."""
+        config = AstrometryTask.ConfigClass()
+        config.doMagnitudeOutlierRejection = True
+        config.magnitudeOutlierRejectionNSigma = 4.0
+        solver = AstrometryTask(config=config, refObjLoader=None)
+
+        nTest = 100
+
+        refSchema = lsst.afw.table.SimpleTable.makeMinimalSchema()
+        refSchema.addField('refFlux', 'F')
+        refCat = lsst.afw.table.SimpleCatalog(refSchema)
+        refCat.resize(nTest)
+
+        srcSchema = lsst.afw.table.SourceTable.makeMinimalSchema()
+        srcSchema.addField('srcFlux', 'F')
+        srcCat = lsst.afw.table.SourceCatalog(srcSchema)
+        srcCat.resize(nTest)
+
+        np.random.seed(12345)
+        refMag = np.full(nTest, 20.0)
+        srcMag = np.random.normal(size=nTest, loc=0.0, scale=1.0)
+
+        # Determine the sigma of the random sample
+        zp = np.median(refMag[: -4] - srcMag[: -4])
+        sigma = scipy.stats.median_abs_deviation(srcMag[: -4], scale='normal')
+
+        # Deliberately alter some magnitudes to be outliers.
+        srcMag[-3] = 4.1*sigma + (20.0 - zp)
+        srcMag[-4] = -4.1*sigma + (20.0 - zp)
```

Comment:
Please make the connection between the `4.1` here, and the `magnitudeOutlierRejectionNSigma = 4.0` above explicit. Maybe by making this multiplier `magnitudeOutlierRejectionNSigma + 0.1`?